src/main/kotlin/interpreter/pyobject/PyObject.kt

```diff
@@ -186,11 +189,5 @@ abstract class PyObject() {
      * The internal [`__dict__`][PyDict] of this object, wrapped. This corresponds to `__dict__`.
      */
     val pyDict: PyDict
-        get() {
-            return PyDict(internalDict.mapKeys {
-                PyString(
-                    it.key
-                )
-            }.toMutableMap() as LinkedHashMap)
-        }
+        get() = PyDict(internalDict.mapKeys { PyString(it.key) }.toMutableMap() as LinkedHashMap)
```

Comment:
`PyDict(internalDict.mapKeysTo(linkedMapOf()) { PyString(it.key) })` saves a toMutableMap call and a LinkedHashMap cast.